xapian-applications/omega/templates/query

```diff
@@ -88,6 +88,7 @@ $list{$map{$queryterms,$list{$html{$uniq{$unstem{$_}}},<b>,</b>/<b>,</b>}:&nbsp;
 <table>
 $if{$opt{clicklog},$set{queryid,$hash{md5,$transform{s/^\s+|\s+$$//g,\1,$query}}}}
 $hitlist{<tr><td valign=top>
+$if{$opt{clicklog},$set{docurl,$field{url}}}
```

Comment:
Um, I meant put the URL that the click should lead in an option - i.e. `$field{url}` in the standard case, and the URL which bounces the user via omega with the `clicklog` template in the click logging case.  As is, you've still got two copies of the code to build the `clicklog` URL in the template, which is harder to maintain (it's all too easy to only update one version), and which makes that part of the template harder to read.

Just changing `$field{url}` to `$opt{docurl}` makes little sense - the latter is no clearer, and there's also the extra code to set it.